projects/js-packages/idc/components/safe-mode/index.jsx

```diff
@@ -84,9 +85,13 @@ const SafeMode = props => {
 		clearErrorType,
 		hasError = false,
 		customContent,
+		isDevelopmentSite,
 	} = props;
 	const [ isStayingSafe, setIsStayingSafe ] = useState( false );
 
+	const buttonLabel =
+		customContent.stayInSafeModehButtonLabel || __( 'Stay in Safe mode', 'jetpack' );
```

Comment:
I was trying to understand the `stayInSafeModehButtonLabel` - it looks like it should be defined in `custom-content-shape.jsx`, as other `customContent` values are?

Reply:
Good catch. Fixed in https://github.com/Automattic/jetpack/pull/37023/commits/3fbb75e4fad0036df6dc7934332bbd0f092eb075